client/vnctl/lib/vnctl/cli/network.rb

```diff
@@ -3,19 +3,22 @@
 module Vnctl::Cli
   class Network < Base
     namespace :networks
-    api_suffix "networks"
+    api_suffix 'networks'
+
+    add_shared_options {
+      option :mode, :type => :string, :desc => 'Can be either physical or virtual.'
+      option :network_mode, :type => :string, :desc => 'Can be either physical or virtual.'
```

Comment:
I take it network_mode is now deprecated and only here so older scripts don't break right away? Doesn't need to happen right away but the integration test dataset files should probably be changed to match sooner rather than later.